src/drivers/gitlab.js

```diff
@@ -221,18 +221,35 @@ class Gitlab {
     const endpoint = `/runners?per_page=100`;
     const runners = await this.request({ endpoint, method: 'GET' });
     return await Promise.all(
-      runners.map(async ({ id, description, active, online }) => ({
+      runners.map(async ({ id, description, online }) => ({
         id,
         name: description,
         labels: (
           await this.request({ endpoint: `/runners/${id}`, method: 'GET' })
         ).tag_list,
-        online,
-        busy: active && online
```

Comment:
Doesn't `busy` condition work as intended?

Reply:
No. Indeed active is deprecated and unfortunately never meant "executing"